targets/targets.json

```diff
@@ -8279,6 +8279,32 @@
         "detect_code": ["7012"],
         "release_versions": ["5"]
     },
+    "TI": {
+        "inherits": ["Target"],
+        "public": false,
+        "extra_labels": ["TI"]
+    },
+    "CC32XX": {
+        "inherits": ["TI"],
+        "public": false,
+        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
+        "release_versions": ["5"]
+    },
+    "CC3220SF": {
+        "inherits": ["CC32XX"],
+        "macros": ["MBED_MPU_CUSTOM"],
+        "core": "Cortex-M4",
+        "device_name": "CC3220SF"
+    },
+    "CC3220SF_LAUNCHXL": {
+        "inherits": ["CC3220SF"],
+        "components_add": ["SD", "FLASHIAP"],
+        "device_has": ["USTICKER", "SERIAL", "SERIAL_FC", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "INTERRUPTIN", "SPI", "ANALOGIN", "FLASH", "TRNG", "RTC"],
```

Comment:
We don't implement sleep or lpticker?

Reply:
Sleep has not been implemented. I don't think it's a required feature in Mbed OS, but if there's a demand down the road, we can add it.
I implemented LPTICKER code but the long latency in their HW design makes it useless in Mbed OS.